assignments/objects.js

```diff
@@ -18,27 +18,70 @@ const example = {
 }
 
 // Write your intern objects here:
-
-
+const InternList = [
```

Comment:
InternList is a variable so it should be camelCase and not Pascal Case.  Pascal Case should be used for class/Object constructors and not the object itself.